application-templates/javascript/connect.yaml

```diff
@@ -54,4 +54,15 @@ deployAs:
         - key: CTP_CLIENT_SECRET
           description: commercetools Composable Commerce client secret
         - key: CTP_SCOPE
-          description: commercetools Composable Commerce client scope
\ No newline at end of file
+          description: commercetools Composable Commerce client scope
+  - name: mc-app
+    applicationType: mc-app
+    endpoint: /
+    configuration:
+      standardConfiguration:
+        - key: APPLICATION_ID
+          description: commercetools Composable Commerce application ID
```

Comment:
We can set the description similar to Merchant Center to make it clearer.
`The Custom Application ID is provided when you add a Custom Application in the Merchant Center`
https://docs.commercetools.com/custom-applications/api-reference/application-config#envproductionapplicationid

Reply:
Changed it :)